src/solver/conv_winoRxS.cpp

```diff
@@ -190,31 +190,41 @@ inline bool IsShaderContraintsMet(const ExecutionContext& ctx,
         if(!(0 <= problem.GetBackwardPadH() && problem.GetBackwardPadH() < std::pow(2, 16)))
             return false;
     }
-    const auto grid_workgroup_count_x = ctx.GetStream().GetMaxHardwareComputeUnits();
```

Comment:
@johnny-keker now tidy complaints that `ctx` is not used anywhere

```
[2022-10-14T17:46:36.761Z] /var/jenkins/workspace/MIOpen_wino-applicability-update/src/solver/conv_winoRxS.cpp:172:59: error: unused parameter 'ctx' [-Werror,-Wunused-parameter]

[2022-10-14T17:46:36.762Z] inline bool IsShaderContraintsMet(const ExecutionContext& ctx,

[2022-10-14T17:46:36.762Z]                                                           ^

Reply:
[Side note] @johnny-keker Please keep an eye on CI until all tests passed.